EIPS/eip-4955.md

```diff
@@ -0,0 +1,85 @@
+---
+eip: 4955
+title: Non-Fungible Token Metadata Standard Extension
+description:  A metadata standard extension schema for non-fungible tokens
+author: Ignacio Mazzara (@nachomazzara)
+discussions-to: https://ethereum-magicians.org/t/eip-4955-non-fungible-token-metadata-namespaces-extension/8746
+type: Standards Track
+category: ERC
+status: Draft
+requires: 721, 1155
+created: 2022-03-29
+---
+
+## Abstract
+
+The following standard allows for the implementation of a standard schema for NFTs metadata.
+
+## Motivation
+
+A standard schema allows wallets, marketplaces, metaverses, etc. to work with any NFT.
+
+This standard is inspired by the projects like marketplaces and metaverses which want to leverage the usage of NFTs by rendering them using custom 3d representations or any other new attributes.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
+
+**Proposed Schema** (subject to "caveats" below):
+
+```json
+{
+    "title": "Asset Metadata",
+    "type": "object",
+    "properties": {
+        "name": {
+            "type": "string",
+            "description": "Identifies the asset to which this NFT represents"
+        },
+        "description": {
+            "type": "string",
+            "description": "Describes the asset to which this NFT represents"
+        },
+        "image": {
+            "type": "string",
+            "description": "A URI pointing to a resource with mime type image/* representing the asset to which this NFT represents. Consider making any images at a width between 320 and 1080 pixels and aspect ratio between 1.91:1 and 4:5 inclusive."
+        },
+        "namespaces": {
+          "type": "object",
+          "description": "Projects that needs specific properties to use the NFT"
+        }
+    }
+}
+```
+
+**Example**
+```json
+{
+    "name": "My NFT",
+    "description": "NFT description",
+    "image": "ipfs://QmZfmRZHuawJDtDVMaEaPWfgWFV9iXoS9SzLvwX76wm6pa",
+    "namespaces": {
+      "myAwesomeCompany": {
+        "prop1": "value1",
+        "prop2": "value2",
+      },
+      "myAwesomeCompany2": {
+        "prop3": "value3",
+        "prop4": "value4",
+      },
+    }
+  }
+```
+
+## Rationale
+
+There are many projects which need custom properties in order to display a current NFT. Each project may have its own way to render the NFTs and therefore they need different values. An example of this is the metaverses like Decentraland or TheSandbox where they need different 3d models to render the NFT based on the visual/engine of each. NFTs projects like Cryptopunks, Bored Apes, etc. can create the 3d models needed for each project and therefore be supported out of the box.
```

Comment:
This reads more like it should go in the "Motivation" section.

The "Rationale" section is for describing the reasons behind choices within the EIP itself, while the "Motivation" section is for describing why the EIP as a whole is necessary.

For example, you might describe why the `namespaces` key is a plain object, and not something more structured.

Reply:
Better, but I think it's still too focused on answering "why do we need this standard?".

Good questions to answer in the rationale might be:

 - Why does this standard use vendor names and not MIME types?
 - Why modify `ERC721Metadata` instead of adding a new function?